samples/boards/cc3220sf_launchxl/echo/README.rst

```diff
@@ -0,0 +1,55 @@
+.. _echo_sample:
+
+Socket Echo Sample over CC3220SF WiFi
+#####################################
+
+Overview
+********
+
+This sample reimplements the networking sockets echo sample, leveraging
+the CC3220SF WiFi host driver which is part of the SimpleLink SDK.
+
+This is based on the Zephyr sockets echo sample, with the following
```

Comment:
How about adding a link to the other sample:

    This Wi-Fi sample is based on the 
    :ref:`Zephyr sockets echo sample <sockets-echo-sample>`, with the following

Reply:
OK.